docs/BUILD.md

```diff
@@ -181,6 +181,116 @@ sudo dnf install openssl-devel
 sudo dnf install libatomic
 ```
 
+#### Linux XDP
+Linux XDP is experimentally supported on amd64 && Ubuntu 22.04LTS.  
+Commands below automatically install dependencies and setup runtime environment.
+```sh
+pwsh ./scripts/prepare-machine.ps1 -UseXdp
+pwsh ./scripts/build.ps1
+export MSQUIC_ENABLE_XDP=1
+```
+
+`./scripts/prepare-machine.ps1` internally donig belows
+```sh
+# for libxdp v1.4.2
+sudo apt-add-repository "deb http://mirrors.kernel.org/ubuntu noble main" -y
+
+# install runtime dependencies
+sudo apt-get install -y libxdp1 libbpf1 libnl-3-200 libnl-route-3-200 libnl-genl-3-200
+
+# install build dependencies
+sudo apt-get --no-install-recommends -y install libxdp-dev libbpf-dev libnl-3-dev libnl-genl-3-dev libnl-route-3-dev zlib1g-dev zlib1g pkg-config m4 clang libpcap-dev libelf-dev libc6-dev-i386
+
+# Optional. This is required when you run test with duonic (XDP capable virtual nic pair)
+sudo apt-get -y install iproute2 iptables
+sudo ./scripts/duonic.sh install
+```
+
+Test
+```sh
+# "sudo" and MSQUIC_ENABLE_XDP=1 required
+# You can explicitly specify directory of datapath_raw_xdp_kern.o by MSQUIC_XDP_OBJECT_PATH
+# By default, libmsquic.so searchs for same directory as its executable
+# If something failed, fallback to normal socket
+sudo MSQUIC_ENABLE_XDP=1 ./artifacts/bin/linux/x64_Debug_openssl3/msquictest --duoNic
+```
+
+**Q&A**
+- Q: Several stderr printed. Is this okey?  
+A: It is ignorable error coming from libbpf.so
```

Comment:
There's got to be a way to suppress these. We can't have our library just spit out random console messages.

Reply:
No, The error `libbpf: elf: skipping unrecognized data section(7) xdp_metadata` comes from libbpf to open `xdp-dispathcer.o`. `xdp-dispathcer.o` is part of libxdp. Both packaged and built from source shows the error. Need libbpf to fix.
LIBXDP_SKIP_DISPATHCER=1 can avoid opening xdp-dispathcer.o, but There are another error from libxdp
```
libxdp: Couldn't find a BPF file with name xdp-dispatcher.o
libxdp: Couldn't open BPF file xdp-dispatcher.o
```

Reply:
made PR
https://github.com/xdp-project/xdp-tools/pull/418

Reply:
I just took a look. Is there a way for you to disable `pr_warn` prints to the console?

Reply:
There is no runtime flag to adjust logging.

Reply:
oh, found the one!!!
libxdp_set_print(NULL);
libbpf_set_print(NULL);